### PR TITLE
[vehicle_info_util] add readme

### PR DESCRIPTION
**Pull request by taikitanaka3**
## Description

add readme

## Review Procedure

see readme.md

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Read [pull request guidelines][pull-request-guidelines]
- [x] Code follows [coding guidelines][coding-guidelines]
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: vehicle/util/vehicle_info_util/Readme.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Vehicle Info Util
+
+## Purpose
+
+This package is to get vehicle info from parameter server.
+
+### Discription
```
Comment (tkimura4) [inline vehicle/util/vehicle_info_util/Readme.md:7]: Description?

Comment (taikitanaka3) [inline vehicle/util/vehicle_info_util/Readme.md:7]:
```suggestion
### Description
```

Comment (taikitanaka3) [reply vehicle/util/vehicle_info_util/Readme.md]: 4ca08f1

File: sensing/preprocessor/pointcloud/pointcloud_preprocessor/docs/OccupancyGridMapOutlierFilter.md [hunk 2]
```diff
@@ -83,7 +83,9 @@ Example:
 
 ## (Optional) References/External links
 
+<!--
 - [RadiusSearchOutlier](https://vml.sakura.ne.jp/koeda/PCL/tutorials/html/remove_outliers.html)
+-->
```
Comment (kenji-miyake) [inline sensing/preprocessor/pointcloud/pointcloud_preprocessor/docs/OccupancyGridMapOutlierFilter.md:88]: this link?
https://pcl.readthedocs.io/projects/tutorials/en/latest/remove_outliers.html

Comment (taikitanaka3) [reply sensing/preprocessor/pointcloud/pointcloud_preprocessor/docs/OccupancyGridMapOutlierFilter.md]: I remove comment out commit  from this PR

**Comment by tkimura4** [comment]
@taikitanaka3 
Is it mistakes?
cb43881

**Comment by taikitanaka3** [comment]
@tkimura4 
since it requires message type adaption I try to fixed it at cb43881
image

File: sensing/preprocessor/pointcloud/pointcloud_preprocessor/docs/OccupancyGridMapOutlierFilter.md (hunk 2 above)
Comment (kenji-miyake) [reply sensing/preprocessor/pointcloud/pointcloud_preprocessor/docs/OccupancyGridMapOutlierFilter.md]: Could you replace the old link with this link?

Comment (taikitanaka3) [reply sensing/preprocessor/pointcloud/pointcloud_preprocessor/docs/OccupancyGridMapOutlierFilter.md]: @kenji-miyake 
Done a9e9417

**Comment by taikitanaka3** [comment]
fixed at https://github.com/tier4/AutowareArchitectureProposal.iv/pull/564

**Review by tkimura4 — APPROVED** [review]
LGTM

**Pull request merged**